fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -1171,6 +1172,9 @@ public static MaterializedViewRewriteMode parse(String str) {
     @VarAttr(name = CBO_PRUNE_JSON_SUBFIELD_DEPTH, flag = VariableMgr.INVISIBLE)
     private int cboPruneJsonSubfieldDepth = 20;
 
+    @VarAttr(name = CBO_USE_HISTOGRAM_EVALUDATE_LIST_PARTITION)
```

Comment:
INVISIBLE ？

Reply:
default true ?

Reply:
our histogram is table level and it was manually collected. no guarantee timeliness. it may not be accurate. so we disable it as default.